reviews/2020-brigade-annual.md

```diff
@@ -0,0 +1,85 @@
+    - Since 1.0, we have seen [new contributor PRs drop slightly](https://brigade.devstats.cncf.io/d/52/new-contributors-table?orgId=1&from=now-1y&to=now) compared to the [year prior](https://brigade.devstats.cncf.io/d/52/new-contributors-table?orgId=1&from=1522130400000&to=1553666400000). We believe this is due to the stability and feature-set of 1.0 and are expecting this trend will reverse when we open up feature development for Brigade 2.0. 
+
+
+- *How many maintainers do you have, and which organisations are they from? (Feel free to link to an existing MAINTAINERS file if appropriate.)*
+    - Brigade currently has [7 core maintainers](https://github.com/brigadecore/brigade/blob/master/CODEOWNERS#L4):
+        - Matt Butcher, Microsoft
+        - Adam Reese, Microsoft
+        - Yusuke Kuoka, Z Lab
+        - Radu Matei, Microsoft
+        - Luke Philips, Charter Communications
+        - Kent Rancourt, Microsoft
+        - Vaughn Dice, Microsoft
+
+- *What do you know about adoption, and how has this changed since your last review / since you joined Sandbox? If you can list companies that are end users of your project, please do so. (Feel free to link to an existing ADOPTERS file if appropriate.)*
+    - According to the CNCF Annual Survey 2019, over 40% of respondents have used Brigade
+    - Core Adopters are:
+        - Charter
+        - Z Lab (Yahoo Japan)
+        - iBotta
+        - Microsoft
+        - Student.com
+        - Spotahome.com
+        - ThredUp
+
+- *How has the project performed against its goals since the last review? (We won't penalize you if your goals changed for good reasons.)*
+    - This is Brigade's first annual review, so we didn't appear to have official goals set.  However, after reaching the 1.0 milestone, our goals had been to:
+        - Maintain a relatively consistent feature release cadence
+        - Improve documentation and increase tutorials/examples ([v1.3.0](https://github.com/brigadecore/brigade/releases/tag/v1.3.0) heavily centered around this)
+        - Attend to bug reports and security patches in a timely manner 
+        - Divert some full-time resources towards 2.0
+
+    - On these counts, we consider ourselves successful
+
+- *What are the current goals of the project? For example, are you working on major new features? Or are you concentrating on adoption or documentation?*
+  - Make significant strides towards Brigade 2.0
+  - Improve notes and awareness for Brigade releases
+  - Increase rate of new contributors
+  - We want to set up Brigade to move into incubation in 2022.
+
+- *How can the CNCF help you achieve your upcoming goals?*
+    - We believe opportunities like occasional CNCF webinars or blog posts are good for informing the CNCF user base of our project's features and goals.
+    - When the 2.0 planning cycle ends and development begins, we feel that we need to develop a platform for reaching new contributors.
```

Comment:
Is there an approximate ETA of 2.0 planning cycle completion, and the development kickoff?

Reply:
Anticipated timeline added in-line to this section of the document.  Thank you!